include/cpr/status_codes.h

```diff
@@ -0,0 +1,71 @@
+#ifndef _CPR_STATUS_CODES
+#define _CPR_STATUS_CODES
+#include <cstdint>
+namespace cpr {
+namespace status {
+constexpr std::uint32_t HTTP_100_CONTINUE = 100;
+constexpr std::uint32_t HTTP_101_SWITCHING_PROTOCOL = 101;
```

Comment:
I'm finding it a little unusual to include the code number in the constant name, are you okay with removing it (e.g. make this HTTP_SWITCHING_PROTOCOL)?

Reply:
I had them like that because I was used to Django but I've removed them now.